src/src/AppBundle/Form/RegistrationForm.php

```diff
@@ -15,21 +15,22 @@ class RegistrationForm extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('first_name', TextType::class)
-            ->add('last_name', TextType::class)
-            ->add('email', EmailType::class)
+            ->add('first_name', TextType::class, array('label'=>$options['data']['lang']['first_name']))
```

Comment:
Si si istá že tam ma byt aj to lang?

Reply:
ano. som to otestovala. lebo tam kde to posielam, to posielam ako 
```
array(..., 
'lang' => $contextLang
,...)
```